accounts/manager.go

```diff
@@ -46,8 +46,8 @@ var (
 // Account represents a stored key.
 // When used as an argument, it selects a unique key file to act on.
 type Account struct {
-	Address common.Address // Ethereum account address derived from the key
-	EncryptedKey string // web3JSON format
+	Address      common.Address // Ethereum account address derived from the key
```

Comment:
I think if we go-fmt at the end, then we should be good to go. I just ran go fmt on accounts/* and that was applied to many files.